mace/data/atomic_data.py

```diff
@@ -201,6 +210,30 @@ def from_config(
             if config.charges is not None
             else None
         )
+        if (element_group_weights is None and atom_group_weights is None):
+            peratom_weights = torch.ones(len(config.positions), dtype=torch.get_default_dtype())
+        elif (element_group_weights is not None and atom_group_weights is not None):
+            raise RuntimeError('atom_group_weights and element_group_weights should not be specified at the same time!')
+        elif (atom_group_weights is not None and config.group is None):
+            raise RuntimeError('atom_group_weights is specified but can not find group definitions in configuration!')
+        elif (atom_group_weights is not None):
+            group_list = config.group.tolist()
+            peratom_weights = torch.ones(len(config.positions), dtype=torch.get_default_dtype())
+            for i in set(config.group):
+                if (i not in atom_group_weights.keys()):
+                    raise RuntimeError('can not find weight for group {:d}!'.format(i))
+                weight_avg = atom_group_weights[i] / group_list.count(i)
```

Comment:
regarding the normalization of the weights, if we have a dictionary like: {'Fe':1, 'N':1}

the actual weights will be 1/num(Fe) and 1/num(N), and then later normalized to sum to 1, right? so to give the same weights to each chemical species we should know the (average) number of atoms

Reply:
> the actual weights will be 1/num(Fe) and 1/num(N), and then later normalized to sum to 1, right? so to give the same weights to each chemical species we should know the (average) number of atoms

Yes exactly, we could do this in some other manner, but I feel that there are trade-offs in the ease of use.